doc/design/paths-endpoint-webdav.md

```diff
@@ -0,0 +1,21 @@
+# Discussion on metadata needed for dandidav
+
+Submitted as a PR to avoid spagetti thread discussion on https://github.com/dandi/dandi-archive/issues/1837#issuecomment-1921864949
```

Comment:
```suggestion
Submitted as a PR to avoid spaghetti thread discussion on https://github.com/dandi/dandi-archive/issues/1837#issuecomment-1921864949
```

Reply:
oh -- codespell didn't pick it up!? ;) thanks @jwodder